pkg/common/common.go

```diff
@@ -20,55 +20,55 @@ func GetEnvOrDefault(envName string, defaultValue string) (output string) {
 	return output
 }
 
-// GetAppEnvFile ...
+// GetAppEnvFile -
```

Comment:
Why a hyphen?

Perhaps, if I put all of the comments on the same line this will not be needed?

Reply:
oh, the hyphen was to offer a replacement character for the three dots, as i assumed you were using them for typographical/formatting reasons. They can be removed easy enough, but the three dots for whatever reasons marks the end of a go comment within go's tooling.

Reply:
> the three dots for whatever reasons marks the end of a go comment within go's tooling

I didn't know that! But that explains why in the LSP, it may not show the rest of the info about the comment!